fs/unstable_remove_test.ts

```diff
@@ -0,0 +1,153 @@
+// Copyright 2018-2025 the Deno authors. MIT license.
+
+import { assert, assertRejects, assertThrows } from "@std/assert";
+import { rmSync, writeFileSync } from "node:fs";
+import { rm, writeFile } from "node:fs/promises";
+import { join } from "node:path";
+import { NotFound } from "./unstable_errors.js";
+import { makeTempDir, makeTempDirSync } from "./unstable_make_temp_dir.ts";
+import { remove, removeSync } from "./unstable_remove.ts";
+import { stat, statSync } from "./unstable_stat.ts";
+
+Deno.test("remove() remove an existed and empty directory", async () => {
+  const tempDir = await makeTempDir({ prefix: "remove_async_" });
+  const existedCheck = await stat(tempDir);
+  assert(existedCheck.isDirectory === true);
+
+  try {
+    await remove(tempDir);
+    const existedCheckAgain = await stat(tempDir);
+    assert(existedCheckAgain.isDirectory === false);
+  } catch {
```

Comment:
This needs to be `finally`. Currently this catches the error thrown from `await stat(tempDir)` and swallows it (The last assertion doesn't seem working).

Reply:
Updated.

Thank you~